compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/FirSourceChild.kt

```diff
@@ -9,16 +9,14 @@ import com.intellij.psi.tree.IElementType
 import com.intellij.psi.tree.TokenSet
 import org.jetbrains.kotlin.fir.*
 
-fun FirSourceElement.getChild(type: IElementType, index: Int = 0, depth: Int = -1): FirSourceElement? {
-    return getChild(setOf(type), index, depth)
-}
+fun FirSourceElement.getChild(type: IElementType, index: Int = 0, depth: Int = -1): FirSourceElement? =
```

Comment:
Why did you remove `{}`? IMO function with actual body with `return` statement is much cleaner that single expression body in almost all cases

Reply:
Oh, it was a remnant from my first approach to finding a previous sibling where I utilize get*. Let me revert this in a follow-up PR.

Reply:
Sent out https://github.com/JetBrains/kotlin/pull/4178